recipes/libjwt/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conan import ConanFile, conan_version
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get
+from conan.tools.system import package_manager
+from conan.tools.scm import Version
+
+
+class libjwtRecipe(ConanFile):
+    name = "libjwt"
+    package_type = "library"
+
+    # Optional metadata
+    license = "Mozilla Public License Version 2.0"
+    author = "Ben Collins"
+    url = "https://github.com/benmcollins/libjwt"
+    description = "The C JSON Web Token Library +JWK +JWKS"
+    topics = ("json", "jwt", "jwt-token")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    def source(self):
+        get(self, self.conan_data["sources"][self.version]["url"], strip_root=True)
+
+    def requirements(self):
+        self.requires("openssl/3.3.2")
+        self.requires("jansson/2.14")
+
+    def system_requirements(self):
+        apt = package_manager.Apt(self)
+        apt.install(["libjansson-dev"], update=True, check=True)
+        
+        yum = package_manager.Yum(self)
+        yum.install(["jansson-devel"], update=True, check=True)
+
+        dnf = package_manager.Dnf(self)
+        dnf.install(["jansson-devel"], update=True, check=True)
+
+        zypper = package_manager.Zypper(self)
+        zypper.install(["libjansson-devel"], update=True, check=True)
+
+        pacman = package_manager.PacMan(self)
+        pacman.install(["jansson"], update=True, check=True)
+
+        pkg = package_manager.Pkg(self)
+        pkg.install(["jansson"], update=True, check=True)
```

Comment:
@AbrilRBS when creating this PR we had a doubt about how best to handle this dependency. We saw that it is already in Conan but we couldn't find a better way to satisfy current `libjwt` build process than by installing a system requirement. I saw that you managed to avoid a system dependency in your PR but it's not clear to me if the same can be attempted with v1.

Reply:
Yes, the same approach should work for both versions, as Conan is already managing the necessary dependencies in the respective recipes - if any of the required configurations fail, we can take a look at that specific case, but usually just using normal Conan recipes for dependencies is much more desired